specification/security/resource-manager/Microsoft.Security/preview/2021-07-01-preview/customEntityStoreAssignment.json

```diff
@@ -0,0 +1,335 @@
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/customEntityStoreAssignmentName"
+          },
+          {
+            "$ref": "#/parameters/customEntityStoreAssignmentRequestBody"
+          }
+        ],
+        "x-ms-examples": {
+          "Create a custom entity store assignment": {
+            "$ref": "./examples/CustomEntityStoreAssignments/customEntityStoreAssignmentCreate_example.json"
+          }
+        },
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/CustomEntityStoreAssignment"
+            }
+          },
+          "200": {
+            "description": "OK - Updated",
+            "schema": {
+              "$ref": "#/definitions/CustomEntityStoreAssignment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Deleted a custom entity store assignment",
+        "description": "Delete a custom entity store assignment by name for a provided subscription",
+        "tags": [
+          "customEntityStoreAssignments"
+        ],
+        "operationId": "CustomEntityStoreAssignments_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/customEntityStoreAssignmentName"
+          }
+        ],
+        "x-ms-examples": {
+          "Delete a custom entity store assignment": {
+            "$ref": "./examples/CustomEntityStoreAssignments/customEntityStoreAssignmentDelete_example.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK - Custom Entity Store Assignment was deleted"
+          },
+          "204": {
+            "description": "No Content - Custom Entity Store Assignment does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Security/customEntityStoreAssignments": {
+      "get": {
+        "summary": "List custom entity store assignments in a subscription",
```

Comment:
Define x-ms-paging on collection API. Currently even if you don't need/support paging, by declaring the paging constructs, you can enable paging in future with out introducing a new api version.